compatibility-verifier/compCheck.sh

```diff
@@ -287,90 +287,120 @@ fi
 
 # Setup initial cluster with olderCommit and do rolling upgrade
 # Provide abspath of filepath to $COMPAT_TESTER
+echo "Setting up cluster before upgrade"
 startServices "$oldTargetDir"
 
-echo "Setting up cluster before upgrade"
-$COMPAT_TESTER $testSuiteDir/pre-controller-upgrade.yaml 1
-if [ $? -ne 0 ]; then
-  if [ $keepClusterOnFailure == "false" ]; then
-    stopServices
+genNum=0
+if [ -f $testSuiteDir/pre-controller-upgrade.yaml ]; then
```

Comment:
Might be good to list out all the yaml files before executing them?

Reply:
I don't see any gain in that. Perhaps it is useful to echo a message if there is a missing yaml file (e.g. if "pre-controller-upgrade.yaml missing. Not running any tests"). I can include that in the next PR if you think that is useful.